services/w3c/w3c-validation-helper.js

```diff
@@ -0,0 +1,157 @@
+'use strict'
+
```

Comment:
Nice. Extracting these items in here really helps the main service file read more succinctly 👍